ui/components/app/snaps/snap-ui-footer-button/snap-ui-footer-button.tsx

```diff
@@ -90,6 +90,7 @@ export const SnapUIFooterButton: FunctionComponent<
         alignItems: AlignItems.center,
         flexDirection: FlexDirection.Row,
       }}
+      data-testid={`${name}-snap-footer-button`}
```

Comment:
this fixes most of our problems within the selectors of the send actions in solana screen